### DISCUSSION TITLE
Logarithmic scales

### DISCUSSION THREAD

**Discussion by denxorz** [imaNNeo/fl_chart#1439] [Ideas]
Is the feature for logarithmic scales somewhere on the roadmap? Or is there a way to mimic this behaviour our selfs?

**Comment by TJMusiitwa** [comment]
Yes this would be a wonderful addition to the package. As opposed to using mountains of code to work out the right scale and often times it is not the correct one, leading to a messy axis.

Or maybe you have some thoughts on how to implement this temporarily @imaNNeo

**Reply by imaNNeo** [reply]
I have no idea at the moment. You can follow this issue: #434
I will close this discussion though
